### PR TITLE
Add Polish localization - Tutorials

### PR DESCRIPTION
**Pull request by mfilocha**
This is a part of the initial Polish localization - see: https://github.com/kubernetes/website/pull/18419

### CODE REVIEW

**Comment by sftim** [comment]
/hold
pending approval & merge for #18419. Once that merges, it's OK to /unhold
People can /lgtm (for localization review) and /approve, even whilst this is held.

File: content/pl/docs/tutorials/kubernetes-basics/_index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      </div>
+    </div>
+
+    <br>
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>Co Kubernetes może dla Ciebie zrobić?</h2>
+        <p>Użytkownicy oczekują od współczesnych serwisów internetowych dostępności non-stop, a deweloperzy chcą móc instalować nowe wersje swoich serwisów kilka razy dziennie. Używając kontenerów można przygotowywać oprogramowanie w taki sposób, aby mogło być instalowane i aktualizowane łatwo i nie powodując żadnych przestojów. Kubernetes pomaga uruchamiać te aplikacje w kontenerach tam, gdzie chcesz i kiedy chcesz i znajdować niezbędne zasoby i narzędzia wymagane do ich pracy. Kubernetes może działać w środowiskach produkcyjnych, jest otwartym oprogramowaniem zaprojektowanym z wykorzystaniem nagromadzonego przez Google doświadczenie w zarządzaniu kontenerami, w połączeniu z najcenniejszymi idee społeczności.</p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/_index.html]:
```suggestion
        <p>Użytkownicy oczekują od współczesnych serwisów internetowych dostępności non-stop, a deweloperzy chcą móc instalować nowe wersje swoich serwisów kilka razy dziennie. Używając kontenerów można przygotowywać oprogramowanie w taki sposób, aby mogło być instalowane i aktualizowane łatwo i nie powodując żadnych przestojów. Kubernetes pomaga uruchamiać te aplikacje w kontenerach tam, gdzie chcesz i kiedy chcesz i znajdować niezbędne zasoby i narzędzia wymagane do ich pracy. Kubernetes może działać w środowiskach produkcyjnych, jest otwartym oprogramowaniem zaprojektowanym z wykorzystaniem nagromadzonego przez Google doświadczenia w zarządzaniu kontenerami, w połączeniu z najcenniejszymi idee społeczności.</p>

File: content/pl/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><b>Master odpowiada za zarządzanie klastrem.</b> Master koordynuje wszystkie działania klastra, takie jak zlecanie uruchomienia aplikacji, utrzymywanie pożądanego stanu aplikacji, skalowanie aplikacji i instalowanie nowych wersji.</p>
+                <p><b>Węzeł to maszyna wirtualna (VM) lub fizyczny serwer, który jest maszyną roboczą w klastrze Kubernetes.</b> Na każdym węźle działa Kubelet, agent zarządzający tym węzłem i komunikujący się z masterem Kubernetes. Węzeł zawiera także narzędzia do obsługi kontenerów, takie jak Docker lub rkt. Klaster Kubernetes w środowisku produkcyjnym powinien składać się minimum z trzech węzłów.</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>Masters manage the cluster and the nodes are used to host the running applications.</i></p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html]:
```suggestion
                    <p><i>Węzły typu master zarządzają klastrem, pozostałe węzły są wykorzystywane do uruchamiania na nich aplikacji. </i></p>

File: content/pl/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            <div class="col-md-8">
+                <p>Kiedy instalujesz aplikację na Kubernetes, polecasz masterowi uruchomienie kontenera z aplikacją. Master zleca uruchomienie kontenera na węzłach klastra. <b>Węzły komunikują się z masterem przy użyciu Kubernetes API</b>, wystawianego przez mastera. Użytkownicy końcowi mogą korzystać bezpośrednio z Kubernetes API do komunikacji z klastrem.</p>
+
+                <p>Klaster Kubernetes może być zainstalowany zarówno na fizycznych, jak i na maszynach wirtualnych. Aby wypróbować Kubernetes, można też wykorzystać  Minikube.  Minikube to "lekka" implementacja Kubernetes, która tworzy VM na maszynie lokalnej i instaluje prosty klaster składający się tylko z jednego węzła. Minikube jest dostępne na systemy Linux, macOS i  Windows. Narzędzie linii poleceń Minikube obsługuje podstawowe operacje na klastrze, takie jak start, stop, informacje o stanie i usunięcie klastra. Na potrzeby tego samouczka wykorzystamy jednak terminal online z zainstalowanym już wcześniej Minikube.</p>
+
+                <p>Teraz, kiedy już wiesz, co to jest Kubernetes, przejdźmy do samouczka online i uruchommy nasz pierwszy klaster!</p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html]:
```suggestion
                <p>Teraz, kiedy już wiesz, co to jest Kubernetes, przejdźmy do samouczka online i stwórzmy nasz pierwszy klaster!</p>

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Cele</h3>
+                <ul>
+                    <li>Poznać Podamy Kubernetes.</li>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]:
```suggestion
                    <li>Poznać Pody Kubernetes.</li>

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+                    <li>Poznać węzły Kubernetes.</li>
+                    <li>Nauczyć się rozwiązywać problemy z aplikacjami.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h2>Pody Kubernetes</h2>
+                <p>Po stworzeniu Deploymentu w Module <a href="/pl/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>, Kubernetes stworzył <b>Pod</b>, który "przechowuje" instancję Twojej aplikacji. Pod jest obiektem abstrakcyjnym Kubernetes, który reprezentuje grupę jednego bądź wielu kontenerów (takich jak Docker lub rkt) wraz ze wspólnymi zasobami dla tych kontenerów. Zasobami mogą być:</p>
+                <ul>
+                    <li>Współdzielony storage, np. Volumes</li>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]:
```suggestion
                    <li>Współdzielona przestrzeń dyskowa, np. Volumes</li>

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+                    <li>Zasoby sieciowe, takie jak unikatowy adres IP klastra</li>
+                    <li>Informacje służące do uruchamiania każdego z kontenerów ⏤ wersja obrazu dla kontenera lub numery portów, które mają być użyte</li>
+                </ul>
+                <p>Pod tworzy model specyficznego dla aplikacji "wirtualnego serwera" i może zawierać różne kontenery aplikacji, które są relatywnie blisko powiązane. Przykładowo, pod może zawierać zarówno kontener z Twoją aplikacją w Node.js, jak i inny kontener dostarczający dane, które mają być opublikowane przez serwer Node.js. Kontenery wewnątrz poda współdzielą adres IP Address i przestrzeń portów, zawsze są uruchamiane wspólnie w tej samej lokalizacji i współdzielą kontekst wykonawczy na tym samym węźle.</p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]:
```suggestion
                <p>Pod tworzy model specyficznego dla aplikacji "wirtualnego serwera" i może zawierać różne kontenery aplikacji, które są relatywnie blisko powiązane. Przykładowo, pod może zawierać zarówno kontener z Twoją aplikacją w Node.js, jak i inny kontener dostarczający dane, które mają być opublikowane przez serwer Node.js. Kontenery wewnątrz poda współdzielą adres IP i przestrzeń portów, zawsze są uruchamiane wspólnie w tej samej lokalizacji i współdzielą kontekst wykonawczy na tym samym węźle.</p>

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+                    <h3>Podsumowanie:</h3>
+                    <ul>
+                        <li>Pody</li>
+                        <li>Węzły</li>
+                        <li>Główne polecenia Kubectl</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                        <p><i>
+                            Pod to grupa jednego lub wielu kontenerów aplikacji (takich, jak Docker czy rkt) zawierający współdzielony storage (volumes), adres IP address i informacje, jak mają być uruchamiane.
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]:
```suggestion
	                            Pod to grupa jednego lub wielu kontenerów aplikacji (takich, jak Docker czy rkt) zawierający współdzielony storage (volumes), adres IP i informacje, jak mają być uruchamiane.

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2>Węzły</h2>
+                <p>Pod jest uruchamiany na <b>węźle <em>(Node)</em></b>. Węzeł jest maszyną roboczą, fizyczną lub wirtualną, w zależności od klastra. Każdy z węzłów jest zarządzany przez węzeł główny <em>(Master)</em>. Węzeł może zawierać wiele podów. Kubernetes master automatycznie zleca uruchomienie podów na różnych węzłach w ramach klastra. Automatyczne zlecanie uruchomienia bierze pod uwagę zasoby dostępne na każdym z węzłów.</p>
+
+                <p>Na każdym węźle Kubernetes działają co najmniej:</p>
+                <ul>
+                    <li>Kubelet, proces odpowiedzialny za komunikację pomiędzy Kubernetes Master i węzłami; zarządza podami i kontenerami działającymi na maszynie.</li>
+                    <li>Proces wykonawczy kontenera (przykładowo Docker czy rkt), który zajmuje się pobraniem obrazem dla kontenera z repozytorium, rozpakowaniem kontenera i uruchomieniem aplikacji.</li>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]:
```suggestion
                    <li>Proces wykonawczy kontenera (przykładowo Docker czy rkt), który zajmuje się pobraniem obrazu dla kontenera z repozytorium, rozpakowaniem kontenera i uruchomieniem aplikacji.</li>

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+                </ul>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i> Kontenery powinny być uruchamiane razem w jednym podzie, jeśli są ściśle ze sobą związane i muszę współdzielić zasoby, np. dysk.</i></p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]:
```suggestion
                    <p><i> Kontenery powinny być uruchamiane razem w jednym podzie, jeśli są ściśle ze sobą związane i muszą współdzielić zasoby, np. dysk.</i></p>

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_03_nodes.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2>Rozwiązywanie problemów przy pomocy kubectl</h2>
+                <p>W module <a href="/pl/docs/tutorials/kubernetes-basics/deploy-intro/">2</a> używałeś narzędzia Kubectl. W module module 3 będziemy go nadal używać, aby wydobyć informacje na temat zainstalowanych aplikacji i środowiska, w jakim działają. Najczętsze operacje przyprowadzane są przy pomocy następujących poleceń kubectl:</p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]:
```suggestion
                <p>W module <a href="/pl/docs/tutorials/kubernetes-basics/deploy-intro/">2</a> używałeś narzędzia Kubectl. W module 3 będziemy go nadal używać, aby wydobyć informacje na temat zainstalowanych aplikacji i środowiska, w jakim działają. Najczęstsze operacje przeprowadzane są przy pomocy następujących poleceń kubectl:</p>

File: content/pl/docs/tutorials/kubernetes-basics/expose/expose-intro.html [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+
+			<p>Serwis w Kubernetes jest abstrakcyjnym obiektem, która definiuje logiczny zbiór podów oraz politykę dostępu do nich. Serwisy pozwalają na swobodne łączenie zależnych podów. Serwis jest zdefiniowany w YAMLu <a href="/docs/concepts/configuration/overview/#general-configuration-tips">(zalecane)</a> lub or JSONie tak, jak wszystkie obiekty Kubernetes. Zbiór podów, które obsługuje Serwis jest zazwyczaj określany przez <i>LabelSelector</i> (poniżej opisane jest, w jakich przypadkach możesz potrzebować zdefiniować Serwis bez specyfikowania <code>selektora</code>).</p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/expose/expose-intro.html]:
```suggestion
			<p>Serwis w Kubernetes jest abstrakcyjnym obiektem, która definiuje logiczny zbiór podów oraz politykę dostępu do nich. Serwisy pozwalają na swobodne łączenie zależnych podów. Serwis jest zdefiniowany w YAMLu <a href="/docs/concepts/configuration/overview/#general-configuration-tips">(zalecane)</a> lub w JSONie - tak, jak wszystkie obiekty Kubernetes. Zbiór podów, które obsługuje Serwis, jest zazwyczaj określany przez <i>LabelSelector</i> (poniżej opisane jest, w jakich przypadkach możesz potrzebować zdefiniować Serwis bez specyfikowania <code>selektora</code>).</p>

File: content/pl/docs/tutorials/kubernetes-basics/expose/expose-intro.html [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+
+			<p>Mimo, że każdy pod ma swój unikatowy adres IP, te adresy nie są dostępne poza klastrem, o ile nie zostaną wystawione za pomocą Serwisu. Serwis umożliwia aplikacji przyjmować ruch przychodzący. Serwisy mogą być wystawiane na zewnątrz na kilka różnych sposobów, poprzez określenie <code>typu</code> w ServiceSpec:</p>
+			<ul>
+				<li><i>ClusterIP</i> (domyślnie) - Wystawia serwis poprzez wewnętrzny adres IP w klastrze. W ten sposób serwis jest dostępny tylko wewnątrz klastra.</li>
+				<li><i>NodePort</i> - Wystawia serwis na tym samym porcie na każdym z wybranych węzłów klastra przy pomocy NAT. W ten sposób serwis jest dostępny z zewnątrz klastra poprzez <code>&lt;NodeIP&gt;:&lt;NodePort&gt;</code>. Nadzbiór ClusterIP.</li>
+				<li><i>LoadBalancer</i> - Tworzy zewnętrzny load balancer u bieżącego dostawcy usług chmurowych (o ile jest taka możliwość) i przypisuje serwisowi stały, zewnętrzny adres IP. Nadzbiór NodePort.</li>
+				<li><i>ExternalName</i> - Wystawia Service przy pomocy wybranej nazwy (wyspecyfikowanej jako <code>externalName</code>) by returning a CNAME record with the name. No proxy is used. This type requires v1.7 or higher of <code>kube-dns</code>.</li>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/expose/expose-intro.html]:
```suggestion
				<li><i>ExternalName</i> - Wystawia Service przy pomocy wybranej nazwy (wyspecyfikowanej jako <code>externalName</code>) poprzez zwracanie wartości rekordu CNAME przypisanego w usłudze DNS. W tym przypadku nie jest wykorzystywany proces przekierowania ruchu metodą proxy. Ten typ wymaga wersji v1.7 lub wyższej usługi <code>kube-dns</code>.</li>

File: content/pl/docs/tutorials/kubernetes-basics/expose/expose-intro.html [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+			</ul>
+			<p>Więcej informacji na temat różnych typów serwisów znajduje się w  samouczku <a href="/docs/tutorials/services/source-ip/">Using Source IP</a>. Warto też zapoznać się z <a href="/docs/concepts/services-networking/connect-applications-service">Connecting Applications with Services</a>.</p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/expose/expose-intro.html]:
```suggestion
			<p>Więcej informacji na temat różnych typów serwisów znajduje się w  samouczku <a href="/docs/tutorials/services/source-ip/">Używanie adresu źródłowego (Source IP)</a>. Warto też zapoznać się z <a href="/docs/concepts/services-networking/connect-applications-service">Łączeniem Aplikacji z Serwisami</a>.</p>
```

File: content/pl/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 14]
```diff
@@ -0,0 +1,121 @@
...
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_05_scaling1.svg">
+                        </div>
+
+                        <div class="item carousel-item">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_05_scaling2.svg">
+                        </div>
+                      </div>
+
+                      <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+                        <span class="sr-only ">Previous</span>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/scale/scale-intro.html]:
```suggestion
                        <span class="sr-only ">Wstecz</span>

File: content/pl/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 15]
```diff
@@ -0,0 +1,121 @@
...
+                      </a>
+                      <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+                        <span class="sr-only">Next</span>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/scale/scale-intro.html]:
```suggestion
                        <span class="sr-only">Dalej</span>

File: content/pl/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_06_rollingupdates3.svg">
+                        </div>
+
+                        <div class="item carousel-item">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_06_rollingupdates4.svg">
+                        </div>
+                      </div>
+
+                      <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+                        <span class="sr-only ">Previous</span>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/update/update-intro.html]:
```suggestion
                        <span class="sr-only ">Wstecz</span>

File: content/pl/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+                      </a>
+                      <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+                        <span class="sr-only">Next</span>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/update/update-intro.html]:
```suggestion
                        <span class="sr-only">Dalej</span>

File: content/pl/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+
+                    </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>Podobnie, jak w przypadku skalowania aplikacji, jeśli Deployment jest udostępniony publicznie, Serwis będzie kierował ruch tylko do Podów, które są dostępne w trakcie aktualizacji. Dostępny Pod do taki, którego instancja jest dostępna dla użytkowników aplikacji.</p>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/update/update-intro.html]:
```suggesting
                <p>Podobnie, jak w przypadku skalowania aplikacji, jeśli Deployment jest udostępniony publicznie, Serwis będzie kierował ruch tylko do Podów, które są dostępne w trakcie aktualizacji. Dostępny Pod to taki, którego instancja jest dostępna dla użytkowników aplikacji.</p>

File: content/pl/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+
+                <p>Płynne aktualizacje pozwalają na:</p>
+                <ul>
+                    <li>Promocję aplikacji z jednego środowiska do innego (poprzez aktualizację obrazu kontenera)</li>
+                    <li>Wycofywanie się do poprzedniej wersji</li>
+                    <li><em>Continuous Integration</em> oraz <em>Continuous Delivery</em> aplikacji bez przerw w jej działaniu</li>
```
Comment (nvtkaszpir) [inline content/pl/docs/tutorials/kubernetes-basics/update/update-intro.html]: I believe we should add those to the glossary in the future, such as:
Contiuous Integration, Contiuous Delivery,  Continuous Deployment.

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html (hunk 6 above)
Comment (sftim) [reply content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]: How about _adres IP_?

Comment (nvtkaszpir) [reply content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html]: `adres IP` is a valid polish translation of the `IP Address`.

In here `adres IP Address` was just incorrect.

**Pull request merged**